src/main/java/seedu/address/MainApp.java

```diff
@@ -180,4 +184,11 @@ public void stop() {
             logger.severe("Failed to save preferences " + StringUtil.getDetails(e));
         }
     }
+
+    /**
+     * Opens the given {@code String} URL in a browser.
+     */
+    public static void openUrl(String url) {
```

Comment:
Was initially considering implementing this method under `UiManager`, however, that requires the `HostServices` returned from `getHostServices()` be passed to `UiManager`, which I felt was not ideal as it would cause the same instance to be accessed across multiple classes. I settled on having it be implemented here such that if necessary, other windows may open URLs by using the static method. Do let me know your thoughts/comments on possible alternative implementations or locations for this function